src/avram/migrator/alter_table_statement.cr

```diff
@@ -127,6 +127,27 @@ class Avram::Migrator::AlterTableStatement
     {% end %}
   end
 
+  alias POLYMORPHIC_TYPES = Int16.class | Int32.class | Int64.class | UUID.class
```

Comment:
Is this used anywhere?

Reply:
I think it was using before `foreign_key_type` hahahah good catch, thanks!